src/adera/Machines/Rocket.cpp

```diff
@@ -225,6 +197,7 @@ void SysMachineRocket::update_physics(ActiveScene& rScene)
         rCompRb.m_inertiaDirty = true;
 
         // Perform resource consumption calculation
+        /*
```

Comment:
How much work would it take to get this stuff working again in the new system? I suppose it will muddy this PR a bit so I guess putting it off for a later one is fine.

Reply:
A new system (a node update) would be needed for fuel flow which should, yes, go into a different PR. Nodes would be far more capable in handling multiple fuel tanks; I got this nice diagram if I haven't posted it before.
![image](https://user-images.githubusercontent.com/14708882/120094643-67169680-c0d6-11eb-912f-a621928e0f27.png)